shuttle/templates/schedules/shuttle_schedules.html

```diff
@@ -13,7 +13,24 @@ <h6>Use <a href="https://docs.google.com/spreadsheets/d/1VGuZm58MfKW7TBFG09oGT8e
         <iframe width="100%" height=265 src="https://docs.google.com/spreadsheets/d/e/2PACX-1vTy3uE_5hm1xmXyHdlAUwh3UF31yNw_lE1fACWoj1f3ND-UsMsveEe_FDHLLc8pHxgd4YwmdGBqMWRU/pubhtml?widget=true&amp;headers=false"></iframe>
     </div>
     <div class="center-page">
-         <input class="btn btn-warning btn-sm" type="submit" value="Submit to Database">
+         <input id="submit-schedule-button" class="btn btn-warning btn-sm" type="submit" value="Submit to Database">
     </div>
 
+    <script>
+    $(document).ready(function() {
+        $("#submit-schedule-button").click(function () {
+            // Sends google sheets schedule to database (erases old schedule)
+            $.ajax({
+                url: "{{ url_for('SchedulesView:send_schedule_path') }}",
+            }).done(function(data) {
+                alert(data);
+            })
+            .fail(function () {
+                    alert("Something went wrong. Please check that the table formatting is correct or call the " +
```

Comment:
Don't use pop up alerts, they are bad.
https://ux.stackexchange.com/a/72642

Instead use in page alerts. The project is already setup to use them.

In Python when you know something fails call self.sc.set_alert() with the correct parameters (method lives in shuttle controller) and then in javascript when we fail, reload the page and the error will be displayed.

```@route('/create', methods=['POST'])
    def create_new_time_slot(self):
        self.wcc.check_roles_and_route(['Administrator'])
        form = request.form

        start_time = form.get('start-time')
        start_time = datetime.strftime(datetime.strptime(start_time, '%H:%M'), '%H:%M:%S')

        end_time = form.get('end-time')
        end_time = datetime.strftime(datetime.strptime(end_time, '%H:%M'), '%H:%M:%S')

        is_active = form.get('active')

        created = self.sc.create_time_slot(start_time, end_time, is_active)

        if created:
            self.wcc.set_alert('success', 'Schedule Created Successfully!')
        else:
            self.wcc.set_alert('danger', 'Schedule already exists!')
        return redirect(url_for('SchedulesView:create_time_slot'))
```

This is an example of the Python side from https://github.com/betheluniversity/writing-center-python. Feel free to reference it or ask questions.